### PR TITLE
Email setup config #152238709

### PR DESCRIPTION
**Pull request by machariamarigi**
### What does this PR do

- Allow healthchecks to send emails via gmail.

### Description of the task to be completed

- Configure the application to use a gmail account to send emails to users.

### Pivotal Tracker story

- See [ Chore #152238709](https://www.pivotaltracker.com/story/show/152238709)

### CODE REVIEW

File: hc/settings.py [hunk 1]
```diff
@@ -20,9 +20,17 @@
...
 ALLOWED_HOSTS = []
-DEFAULT_FROM_EMAIL = 'healthchecks@example.org'
+DEFAULT_FROM_EMAIL = os.getenv('EMAIL_USER')
 USE_PAYMENTS = False
 
+EMAIL_BACKEND = "djmail.backends.default.EmailBackend"
+DJMAIL_REAL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
+EMAIL_USE_TLS = True
+EMAIL_USE_SSL = False
+EMAIL_HOST = 'smtp.gmail.com'
```
Comment (ivantoz) [inline hc/settings.py]: Lets also have this secured

File: hc/settings.py [hunk 2]
```diff
@@ -20,9 +20,17 @@
...
+EMAIL_HOST_USER = os.getenv('EMAIL_USER')
+EMAIL_HOST_PASSWORD = os.getenv('EMAIL_PASSWORD')
+EMAIL_PORT = 587
```
Comment (ivantoz) [inline hc/settings.py]: Same with this

File: hc/settings.py [hunk 3]
```diff
@@ -124,7 +132,6 @@
 
 COMPRESS_OFFLINE = True
 
-EMAIL_BACKEND = "djmail.backends.default.EmailBackend"
```
Comment (ivantoz) [inline hc/settings.py:127]: What is the reason for removing this line of code?

**Review by ivantoz — CHANGES_REQUESTED** [review]
- Can you address the comments I have made on your code
- Also your PR heading and template does not follow our guidelines
- Your commit messages does not adhere to our standards

File: hc/settings.py (hunk 3 above)
Comment (machariamarigi) [reply hc/settings.py:127]: I moved it up to line 26

**Pull request closed without merging**